portal/tasks.py

```diff
@@ -49,8 +48,6 @@
 #   rdb.set_trace()
 # Follow instructions from celery console, i.e. telnet 127.0.0.1 6900
 
-logger = get_task_logger(__name__)
```

Comment:
I think removing this line is causing an import to break here in [`celery_worker.py`](https://github.com/uwcirg/truenth-portal/blob/v22.5.2.2/portal/celery_worker.py#L27)

[See CI error](https://gitlab.com/uwcirg/truenth-portal/-/jobs/2495593993#L92)